feat(vite-plugin): support different extensions for catalogs. Improve coverage (#1526)
lingui/js-lingui: packages/vite-plugin/README.md

# @lingui/vite-plugin

> > Vite plugin which compiles on the fly lingui catalogs. In summary, `lingui compile` command isn't required when using this plugin

Reviewer: ```suggestion
> Vite plugin that compiles Lingui catalogs on the fly. In summary, the `lingui compile` command isn't needed when using this plugin.
```
